src/settings.py

```diff
@@ -49,11 +49,13 @@ class GlobalSettings:
     behavior_config_file = os.path.join(  # relative to igibson.root_path
         "examples",
         "configs",
-        # "njk_re-shelving_library_books_full_obs.yaml",
-        "njk_sorting_books_full_obs.yaml")
+        "wbm3_modifiable_full_obs.yaml",
+    )
     behavior_mode = "headless"  # headless, pbgui, iggui
     behavior_action_timestep = 1.0 / 10.0
     behavior_physics_timestep = 1.0 / 120.0
+    behavior_task_name = "re-shelving_library_books"
+    behavior_scene_id = "Pomaria_1_int"
```

Comment:
Small thing, but could we change the name of this to something other than `behavior_scene_id`? Really, this is a house/environment/scene name, so could we change it to '<something something> name` instead of 'id` so users don't get confused and think this should be a number?

Reply:
+1

Reply:
behavior_task_name and behavior_scene_name would be good i think?

Reply:
Sure, but originally I kept it as behavior_scene_id because that's what they decided to call it in the .yaml file.

Reply:
oh then maybe we should keep this as is. could you also add a comment describing what are the allowable values for these settings? maybe a link to a yaml file with all the options spelled out?